senior_teacher.rb

```diff
@@ -1,4 +1,11 @@
-class SeniorTeacher
+require_relative 'teachable'
+require_relative 'raiseable'
+require_relative 'apprentice_teacher'
+require 'awesome_print'
+
+class SeniorTeacher < ApprenticeTeacher
```

Comment:
A Senior Teacher is not an Apprentice Teacher.  They are both Teachers.  How can you represent this?